### ISSUE TITLE
Timepicker: Missing Audible Labels

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4403]
**Describe the bug**
Recent VPAT scan indicates our timepicker has some flaws. Its missing labels (maybe a dropdown issue) and has a wrong role.

- role="combobox" is expecting children so is not correct
- there is a label on the select but it isn't read since its attached to the hidden select

```
<div class="dropdown" data-options="{'noSearch': 'true'}" role="button" aria-haspopup="listbox" tabindex="0"><span>1</span></div>
```

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/timepicker/example-index.html
2. Click on the trigger to open
3. Open Axe/Inspect DOM

**Expected behavior**
Fix the role combobox - add a hidden label feature so it says "1 minutes", "30 hours ect"

**Version**
- ids-enterprise: [e.g. v4.32.0]

**Platform**
 - Chrome / Any

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-31955

**Comment by ssaniel** [comment]
Hi @tmcconechy , This is one of the identified High Priority VPAT items. thank you!

**Comment by jbrcna** [comment]
QA Failed
Failing this issue for now. please see notes below. 
https://master-enterprise.demo.design.infor.com/components/timepicker/example-index.html?layout=nofrills
![image](https://user-images.githubusercontent.com/56722184/109445792-59238c00-7a7b-11eb-9807-efb334ade469.png)

https://master-enterprise.demo.design.infor.com/components/datepicker/example-timeformat.html
- Tab / Shift+Tab between the Dropdown inputs (which should now also display visible labels for each field). When focused, each should read their label and currently-selected contents (fx: "Hours, 1" or "Minutes, 00"), followed by their role and state (fx "Button, Collapsed/Expanded")
- it doesn't mention the "Hours", "Minutes", "Seconds", "Period" before the number

**Date Time Picker with Custom Pattern**
- `Hours` is incorrect (default value: it says 1 instead of the value displayed)

**Date Time Picker in Locale (Time to Midnight)**
- `Hours` is incorrect (default value: it says 1 instead of the value displayed)

**Date Time Picker in Locale (Time to Current)**
- `Hours` is incorrect (default value: it says 1 instead of the value displayed)

**Date Time Timestamp**
- `Hours` is incorrect (default value: it says 1 instead of the value displayed)
- `Minutes` is incorrect (default value: it says 00 instead of the value displayed)
- `Seconds` is incorrect (default value: it says 00 instead of the value displayed)

Also, try to change the time on each field then Tab / Shift+Tab between the Dropdown inputs and see errors in vox.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
